lib/rules/no-get.js

```diff
@@ -54,6 +54,13 @@ module.exports = {
     let currentProxyObject = null;
     let currentClassWithUnknownPropertyMethod = null;
 
+    const filename = context.getFilename();
+
+    // Skip mirage directory
+    if (emberUtils.isMirageDirectory(filename)) {
```

Comment:
Since the `no-get` rule would only trigger on `mirage/config.js` as you mentioned, can we actually limit this behavior to only that file? I'd prefer to limit the scope of this as much as possible.

Reply:
Sure thing 👍